bindata/network/ovn-kubernetes/self-hosted/common/monitor-master.yaml

```diff
@@ -23,26 +23,26 @@ spec:
     - openshift-ovn-kubernetes
   selector:
     matchLabels:
-      app: ovnkube-master
+      app: ovnkube-controller
 ---
 apiVersion: v1
 kind: Service
 metadata:
   labels:
-    app: ovnkube-master
-  name: ovn-kubernetes-master
+    app: ovnkube-controller
+  name: ovn-kubernetes-controller
   namespace: openshift-ovn-kubernetes
   annotations:
     service.beta.openshift.io/serving-cert-secret-name: ovn-master-metrics-cert
 spec:
   selector:
-    app: ovnkube-master
+    app: ovnkube-controller
   clusterIP: None
   publishNotReadyAddresses: true
   ports:
   - name: metrics
     port: 9102
     protocol: TCP
-    targetPort: 9102
+    targetPort: 9102  ### TODO it's now 9112
```

Comment:
@ricky-rav I did not try on self-hosted, but I'd think that if we change this, we must also change service / serviceMonitor for ovnkube-master, right?

Reply:
that's an outdated comment, sorry! :)

Reply:
np!